frontend/packages/dev-console/integration-tests/support/step-definitions/monitoring/dashboard-display.ts

```diff
@@ -0,0 +1,140 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+import { monitoringPO } from '../../pageObjects/monitoring-po';
+import { monitoringPage } from '../../pages/monitoring/monitoring-page';
+import { topologyPage } from '../../pages/topology/topology-page';
+import { devNavigationMenu } from '../../constants/global';
+import { navigateTo } from '../../pages/app';
+
+Given('user opened the url of the workload {string} in topology page', (workloadName: string) => {
+  topologyPage.clickWorkloadUrl(workloadName);
+});
+
+Given('user has workloads of all resource types', () => {
+  // TODO: implement step
```

Comment:
not sure if it's fine to commit with todo comments here and below

Reply:
Currently we are implemented only smoke test gherkin scripts,
These step definitions are auto generated  with // TODO: implement step
To identify the un-implemented step, this would be helpful